### PR TITLE
Add Escea fireplace integration

### PR DESCRIPTION
**Pull request by lazdavila**
## Proposed change

Adding integration for Escea fireplaces

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/56039
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2807
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @lazdavila, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19279).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by rytilahti — DISMISSED** [review]
LGTM :+1:

File: source/_integrations/escea.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Escea
+description: Instructions on how to integrate Escea fireplaces with Home Assistant.
+ha_category:
+  - Climate
+ha_release: '2021.10'
```
Comment (MartinHjelmare) [inline source/_integrations/escea.markdown]:
```suggestion
ha_release: 2022.9
```

File: source/_integrations/escea.markdown [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ha_config_flow: true
+ha_codeowners:
+  - '@lazdavila'
+ha_domain: escea
+ha_homekit: true
+ha_platforms:
+  - climate
+---
+
+The `Escea` integration allows monitoring and control of local [Escea](https://escea.com/) fireplaces. These fireplaces are manufactured in New Zealand and sold throughout Australia and New Zealand.
```
Comment (frenck) [inline source/_integrations/escea.markdown]:
```suggestion
The Escea integration allows monitoring and control of local [Escea](https://escea.com/) fireplaces. These fireplaces are manufactured in New Zealand and sold throughout Australia and New Zealand.
```

File: source/_integrations/escea.markdown [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Supported hardware
+
+Support all current Escea fireplaces with the Smart Heat feature, including: DF; DL; DS; and, DX series.
```
Comment (frenck) [inline source/_integrations/escea.markdown]:
```suggestion
Support all current Escea fireplaces with the Smart Heat feature, including the DF, DL, DS, and DX series.
```

File: source/_integrations/escea.markdown [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+Though these fireplaces are supported, they can only be integrated if they were connected via Ethernet when installed (i.e. these fireplaces do not support Wifi).
```
Comment (frenck) [inline source/_integrations/escea.markdown]:
```suggestion
Though these fireplaces are supported, they can only be integrated if they were connected via Ethernet when installed (i.e., these fireplaces do not support Wifi).
```

File: source/_integrations/escea.markdown [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+{% include integrations/config_flow.md %}
+
```
Comment (frenck) [inline source/_integrations/escea.markdown:25]:
```suggestion
```

File: source/_integrations/escea.markdown [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+## Network settings
+
+The Escea system uses UDP over the local network to find and communicate with Escea devices. For this to work properly, UDP port  3300 must be available for broadcasting and messaging fireplaces, and similarly port UDP port 3300 must be available to listen for responses. The integration currently listens on `0.0.0.0` and broadcasts to all broadcast IPv4 local addresses, which is not configurable.
```
Comment (frenck) [inline source/_integrations/escea.markdown]:
```suggestion
The Escea system uses UDP over the local network to find and communicate with Escea devices. For this to work properly, UDP port 3300 must be available for broadcasting and messaging fireplaces, and similarly, UDP port 3300 must be available to listen for responses. The integration currently listens on `0.0.0.0` and broadcasts to all broadcast IPv4 local addresses, which is not configurable.
```

Comment (frenck) [reply source/_integrations/escea.markdown]: Shouldn't it be listening to the interfaces configured in the Home ASsistant network manager?

File: source/_integrations/escea.markdown [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+## Master controller
+
+Fireplace on / off, fan settings and desired temperature settings are supported. The current room temperature as measured by the fireplace is also supported.
```
Comment (frenck) [inline source/_integrations/escea.markdown]:
```suggestion
Fireplace on / off, fan settings, and desired temperature settings are supported. The current room temperature, as measured by the fireplace, is also supported.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/escea.markdown (hunk 6 above)
Comment (lazdavila) [reply source/_integrations/escea.markdown]: Hi Franck, thanks for the review. I don't know about configuring the Network Manager. In my local setup I did not change anything there. I just had a look and there are no network rules allowing or blocking these UDP ports. The integration seems to work fine without any configuration on my local system. If I need to do something, could you please let me know where to look? Alternatively, if you think I should get rid of the comments about what UDP ports it uses (so as not to confuse people) then please let me know.

**Comment by MartinHjelmare** [comment]
Please rebase on latest `next` branch and remove the unrelated commits.

**Comment by MartinHjelmare** [comment]
There are still unrelated commits in the branch. It's ok to start over in a fresh branch and PR if you want.

**Comment by lazdavila** [comment]
Thanks Martin, you are right. I've really messed this one up. I'll start a new branch and create a new PR.

**Pull request closed without merging**